Makefile

```diff
@@ -11,23 +12,23 @@ LIBS = $(filter-out $(MAIN) ,$(OBJS))
 all: main testToken testParser
 
 main: $(LIBS) srcs/main.o
-	$(CPP) $(CPPFLAGS) -o $@ $^
+	$(CPP) $(CPPFLAGS) -o $@ $^ $(LD)
```

Comment:
Most likely, at linking stage, CPPFLAGS is not needed

Reply:
Removed unnecessary flags.